play25-json-tests-sc14/src/main/scala/play/api/libs/json/scalacheck/Depth.scala

```diff
@@ -0,0 +1,24 @@
+package play.api.libs.json.scalacheck
+
+import scala.language.implicitConversions
+
+class Depth private[Depth] (val depth: Int) extends AnyVal with Counted with Proxy {
+  override def self = depth
+  override protected def throwOnNegative(): Nothing = throw new IllegalArgumentException("Depth cannot be negative")
+  @inline override def count: Int = depth
+  def -(that: Depth) = Depth(this.depth - that.depth)
+  def +(that: Depth) = new Depth(this.depth + that.depth)  // no need to validate
```

Comment:
```suggestion
  def +(that: Depth) = Depth(this.depth + that.depth)  // no need to validate
```

Reply:
i don't want to change any code in this PR except scalatest/scalacheck imports, even a minor change like this, just to be consistent with the rest of the library, so in the future if someone does refactoring via global replace for example doesn't leave out anything, as we have some overlapping pieces